<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindPassword</class>
 <widget class="QWidget" name="FindPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>53</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="account_label">
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="email_label">
        <property name="text">
         <string>绑定的邮箱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="phone_label">
        <property name="text">
         <string>绑定的手机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="accout_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="email_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="phone_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="password_line"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="commit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>找回</string>
   </property>
  </widget>
  <widget class="QLabel" name="head_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>找回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
